modules/feegrant/handle_msg.go

```diff
@@ -0,0 +1,47 @@
+package feegrant
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	feegranttypes "github.com/cosmos/cosmos-sdk/x/feegrant"
+	juno "github.com/forbole/juno/v2/types"
+
+	"github.com/forbole/bdjuno/v2/types"
+)
+
+// HandleMsg implements modules.MessageModule
+func (m *Module) HandleMsg(index int, msg sdk.Msg, tx *juno.Tx) error {
+	if len(tx.Logs) == 0 {
+		return nil
+	}
+
+	switch cosmosMsg := msg.(type) {
+	case *feegranttypes.MsgGrantAllowance:
+		return m.HandleMsgGrantAllowance(tx, cosmosMsg)
+	case *feegranttypes.MsgRevokeAllowance:
+		return m.HandleMsgRevokeAllowance(tx, cosmosMsg)
+	}
+
+	return nil
+}
+
+// HandleMsgGrantAllowance allows to properly handle a MsgGrantAllowance
+func (m *Module) HandleMsgGrantAllowance(tx *juno.Tx, msg *feegranttypes.MsgGrantAllowance) error {
+	allowance, err := msg.GetFeeAllowanceI()
+	if err != nil {
+		return fmt.Errorf("error while getting fee allowance: %s", err)
+	}
+	feeGrant, err := feegranttypes.NewGrant(sdk.AccAddress(msg.Granter), sdk.AccAddress(msg.Grantee), allowance)
```

Comment:
Instead of using `sdk.AccAddress(address)` we should use `sdk.AccAddressFromBech32` instead:
```suggestion
	granter, err := sdk.AccAddressFromBech32(msg.Granter)
	if err != nil {
		return fmt.Errorf("error while parsing granter address: %s", err)
	}
	grantee, err := sdk.AccAddressFromBech32(msg.Grantee)
	if err != nil {
		return fmt.Errorf("error while parsing grantee address: %s", err)
	}
	feeGrant, err := feegranttypes.NewGrant(granter, grantee, allowance)
```